docs/advanced_topics/images/feature_detection.rst

```diff
@@ -58,10 +63,55 @@ You can test that OpenCV can be seen by Wagtail by opening up a python shell (wi
 If you don't see an ``ImportError``, it worked. (If you see the error ``libdc1394 error: Failed to initialize libdc1394``, this is harmless and can be ignored.)
 
 
+Rustface
+~~~~~~~~
+
+Rustface_ is Python library with prebuilt wheel files provided for Linux and macOS. Although implemented in Rust it is pip-installable:
+
+ .. code-block:: console
+
+    $ pip install wheel
+    $ pip install rustface
+
+
+Registering with Willow
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Rustface provides a plug-in that needs to be registered with Willow_.
+
+This should be done somewhere that gets run on application startup:
+
+ .. code-block:: python
+
+    from willow.registry import registry
+    import rustface.willow
+
+    registry.register_plugin(rustface.willow)
+
+For example, in an app's AppConfig.onready_.
```

Comment:
I think this method is just `AppConfig.ready`

Reply:
It is indeed even though the link was correct. Fixed by ea95b99. Thanks for the spot.